module/nvpair/nvpair.c

```diff
@@ -3334,6 +3334,9 @@ MODULE_AUTHOR(ZFS_META_AUTHOR);
 MODULE_LICENSE(ZFS_META_LICENSE);
 MODULE_VERSION(ZFS_META_VERSION "-" ZFS_META_RELEASE);
 
+EXPORT_SYMBOL(nvlist_dump_json);
```

Comment:
No need for `EXPORT_SYMBOL` at line number 3337 and 3338, since we are not doing user-space compilation for this file.

Reply:
Ignore this.